src/data/customer/__test__/driver.test.js

```diff
@@ -10,6 +10,9 @@ import {
   deleteDriverScheduleAsync,
   createDriverScheduleAsync,
   updateDriverScheduleAsync,
+  getDriversWithFiltersAsync,
+  getDriverCountsAsync,
+  getDriverRoutesAsync,
 } from '../Driver';
 
```

Comment:
@vdanylov snapshot is failing:

● Test getDriverRoutesAsync › should get getDriverRoutesAsync success response

    expect(value).toMatchSnapshot()
    
    Received value does not match stored snapshot 1.